sky/utils/schemas.py

```diff
@@ -260,3 +261,126 @@ def get_cluster_schema():
             },
         }
     }
+
+
+def get_config_schema():
+    # pylint: disable=import-outside-toplevel
+    from sky.utils import kubernetes_enums
+    return {
+        '$schema': 'https://json-schema.org/draft/2020-12/schema',
+        'type': 'object',
+        'required': [],
+        'additionalProperties': False,
+        'properties': {
+            'spot': {
+                'type': 'object',
+                'required': [],
+                'additionalProperties': False,
+                'properties': {
+                    'controller': {
+                        'type': 'object',
+                        'required': [],
+                        'additionalProperties': False,
+                        'properties': {
+                            'resources': {
+                                k: v
+                                for k, v in get_resources_schema().items()
+                                # Validation may fail if $schema is included.
+                                if k != '$schema'
+                            },
+                        }
+                    },
+                }
+            },
+            'aws': {
+                'type': 'object',
+                'required': [],
+                'additionalProperties': False,
+                'properties': {
+                    'instance_tags': {
+                        'type': 'object',
+                        'required': [],
+                        'additionalProperties': {
+                            'type': 'string',
+                        },
+                    },
+                    'vpc_name': {
+                        'oneOf': [{
+                            'type': 'string',
+                        }, {
+                            'type': 'null',
+                        }],
+                    },
+                    'use_internal_ips': {
+                        'type': 'boolean',
+                    },
+                    'ssh_proxy_command': {
+                        'oneOf': [{
+                            'type': 'string',
+                        }, {
+                            'type': 'null',
+                        }, {
+                            'type': 'object',
+                            'required': [],
+                            'additionalProperties': {
+                                'type': 'string',
+                            }
+                        }]
+                    },
+                }
+            },
+            'gcp': {
+                'type': 'object',
+                'required': [],
+                'additionalProperties': False,
+                'properties': {
+                    'specific_reservations': {
+                        'type': 'array',
+                        'items': {
+                            'type': 'string',
+                            'minItems': 1,
+                            'maxItems': 1,
+                        }
+                    },
```

Comment:
Not sure why, but I can still specify more than 1 `specific_reservations`

Reply:
Yes there's a bug in my schema, thank you for catching this! Also added a test to confirm this.